### PR TITLE
feat(hardhat-ganache): update ganache-core to ganache

### PR DESCRIPTION
**Pull request by LuisUrrutia**
The package hardhat-ganache uses an old version of ganache and I did the update from ganache-core to ganache 7, but I'm having problem with some tests, so I make this PR in order to look for help too

### CODE REVIEW

**Comment by fvictorio** [comment]
We'll have to look at this in more depth, but as a first comment: we use `yarn`, so the `package-lock.json` here shouldn't be included.

File: packages/hardhat-ganache/src/ganache-service.ts [hunk 1]
```diff
@@ -221,12 +279,12 @@ export class GanacheService {
     }
 
     // Test for unsupported commands
-    if (options.accounts !== undefined) {
-      throw new NomicLabsHardhatPluginError(
-        "@nomiclabs/hardhat-ganache",
-        "Config: ganache.accounts unsupported for this network"
-      );
-    }
+    // if (options.wallet && options.wallet.accounts !== undefined) {
```
Comment (alcuadrado) [inline packages/hardhat-ganache/src/ganache-service.ts:282]: Why is this commented out?

File: packages/hardhat-ganache/test/fixture-projects/hardhat-project-with-configs/hardhat.config.ts [hunk 2]
```diff
@@ -4,14 +4,19 @@ export default {
...
+      wallet: {
+        defaultBalance: 9,
+        totalAccounts: 3,
+        mnemonic:
+          "polar velvet stereo oval echo senior cause cruel tube hobby exact angry",
+      },
     },
   },
-  solidity: "0.5.15",
+  solidity: "0.8.11",
```
Comment (alcuadrado) [inline packages/hardhat-ganache/test/fixture-projects/hardhat-project-with-configs/hardhat.config.ts:21]: Keeping it to 0.5.15 makes the ci go faster. Can we revert this change?

**Review by alcuadrado — COMMENTED** [review]
Wow! Nice PR, thanks. I just left a few minor comments.

Can you also remove the pacakge-lock? We use a yarn monorepo, not npm.

**Comment by alcuadrado** [comment]
Also, can you allow us to modify your branch/pr so that we can add a changeset?

**Pull request closed without merging**